### DISCUSSION TITLE
Make ListBox items listen to click on Desktop, long press on Android

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaUI/Avalonia#11071] [Q&A]
I'm using Avalonia 11.0.0-preview6 to create a cross-platform app, Windows desktop + Android.
I'm trying to create the following behavior in a ListBox: 

- On desktop platforms, I want to call a function when the user clicks each ListBox item
- On Android, the same function must be called but after a "long press" over that item

How can I bind those 2 different events, depending on the platform, on the ListBox items? Thanks

**Comment by maxkatz6** [comment]
There are RightTapped and Holding events.
But you probably only need a ContextRequested event which will automatically listen for both + keyboard (Shift+F10 on windows for example).

**Reply by maxkatz6** [reply]
Keep in mind, that your element also needs a non-Null background (Transparent will work) in order to get pointer input.
And if you need a keyboard context menu, content inside of the item should be focusable.

**Reply by afisch-af** [reply]
Holy Moly, you saved me from _so many_ lost hours!
